bin/run

```diff
@@ -4,14 +4,12 @@
 use Symfony\Component\Console\Input\ArgvInput;
 use OpenEuropa\TaskRunner\TaskRunner;
 use Symfony\Component\Console\Output\ConsoleOutput;
-use Composer\Autoload\ClassLoader;
 
+$classLoader = null;
 if (file_exists(__DIR__.'/../vendor/autoload.php')) {
     $classLoader = require __DIR__.'/../vendor/autoload.php';
 } elseif (file_exists(__DIR__.'/../../../autoload.php')) {
     $classLoader = require __DIR__ . '/../../../autoload.php';
-} else {
-    $classLoader = new ClassLoader();
```

Comment:
I would keep this else but, instead, throw an exception if no class loader is found. Task Runner really depends on a class loader now and the run command should fail if none is found.